src/api/mock/rankingMock.ts

```diff
@@ -0,0 +1,39 @@
+import { RankingResponse } from 'api/types';
+
+export const STATS_MOCK: Pick<RankingResponse, 'stats'> = {
+  stats: {
+    Michał: {
+      bike: 10.1,
+      running: 142.7,
+      swimming: 1.2,
+      walking: 74.7,
+      elevation: 1024,
+      other: 12.1,
+      pushUps: 390,
+    },
+    Marcin: {
+      bike: 360.23,
+      running: 42.5,
+      swimming: 2.34,
+      walking: 23.45,
+      elevation: 2048,
+      other: 50.56,
+      pushUps: 265,
+    },
+  },
+};
+
+export const PLAYERS_MOCK: Pick<RankingResponse, 'players'> = {
```

Comment:
Tu i powyżej - bardzo dobrze, że to jest zdefiniowane jako twarde typy, a nie po prostu luźne obiekty. I faktycznie można przez `Pick`. Niemniej, można to wewnątrz `ranking.types.ts` wyodrębnić jako osobne typy, np 
```
export type Player = {
  avatar: string;
  name: string;
  score: number;
}
```
I będzie tutaj mniej pisania, a tam też powinno być ładniej ;)